src/sentry/api/invite_helper.py

```diff
@@ -211,3 +209,33 @@ def accept_invite(self, user=None):
 
         self.handle_success()
         metrics.incr("organization.invite-accepted", sample_rate=1.0)
+
+    def _needs_2fa(self) -> bool:
+        org_requires_2fa = self.om.organization.flags.require_2fa.is_set
+        user_has_2fa = Authenticator.objects.user_has_2fa(self.request.user.id)
+        return org_requires_2fa and not user_has_2fa
+
+    def _needs_email_verification(self) -> bool:
+        organization = self.om.organization
+        if not (
+            features.has("organizations:required-email-verification", organization)
+            and organization.flags.require_email_verification
+        ):
+            return False
+
+        user = self.request.user
+        primary_email_is_verified = (
+            isinstance(user, User)
+            and UserEmail.objects.filter(
+                user=user,
+                email=user.email,
+                is_verified=True,
+            ).exists()
+        )
+        return not primary_email_is_verified
+
+    def get_onboarding_steps(self) -> Dict[str, bool]:
+        return {
```

Comment:
wonder if we could use a NamedTuple or small class to represent the onboarding steps.

Reply:
I'm 👍 for a small data class in general. But I'm unsure what is the most Pythonic style when each instance gets its own function body. This lends itself to a proliferation of singleton classes, the same pattern as `OrganizationComplianceTask` in this PR. Which I don't think is bad exactly, but it's the kind of pattern that lends itself to, like, an abstract `enum` in Java and never seems as clean in Python. Alternatively, we could do something more similar to how it is now: have a series of stand-alone functions and pass each one to the constructor. Anyway, I'm open to suggestion.

In this particular case, I'm inclined to leave it as is under the [Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_%28computer_programming%29).